src/components/Badge.tsx

```diff
@@ -30,11 +31,15 @@ interface BadgeProps {
   rounded?: boolean
 }
 
-const Badge: FC<BadgeProps> = ({ className, children, truncate }) => {
+const Badge: FC<HasTooltip<BadgeProps>> = ({ className, children, truncate, ...props }) => {
```

Comment:
hm, why `...props` here and not directly `dataTip`?

Reply:
besides that, LGTM! 👏

Reply:
Because the property name is "data-tip" unfortunately... A choice made by ReactTooltip.

Reply:
I could do `{ <other properties here>, 'data-tip': dataTip }` I think, if you'd like?

Reply:
I've gone ahead and changed it, I agree the explicitness is much better :slightly_smiling_face: